docs/ref/settings.txt

```diff
@@ -65,9 +65,10 @@ See :doc:`/howto/error-reporting` for more information.
 Default: ``[]`` (Empty list)
 
 A list of strings representing the host/domain names that this Django site can
-serve. This is a security measure to prevent an attacker from poisoning caches
-and triggering password reset emails with links to malicious hosts by submitting
-requests with a fake HTTP ``Host`` header, which is possible even under many
+serve. This is a security measure to prevent HTTP Host header attacks, in which
+an attacker uses malicious HTTP Host header values to inject code, trigger
```

Comment:
I elected to remove that explanation, as the page that is now linked to has an explanation already.